sharding-sql-test/src/main/resources/sql/visitor/dal/show.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one or more
+  ~ contributor license agreements.  See the NOTICE file distributed with
+  ~ this work for additional information regarding copyright ownership.
+  ~ The ASF licenses this file to You under the Apache License, Version 2.0
+  ~ (the "License"); you may not use this file except in compliance with
+  ~ the License.  You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<sql-cases>
+    <sql-case id="show_table_status" value="SHOW TABLE STATUS" db-types="MySQL" />
+    <sql-case id="show_index_with_index_with_table" value="SHOW INDEX FROM t_order" db-types="MySQL" />
+    <sql-case id="show_index_with_indexes_with_table_and_database" value="SHOW INDEXES FROM t_order FROM sharding_db" db-types="MySQL" />
+    <sql-case id="show_index_with_keys_with_database_and_table" value="SHOW KEYS FROM sharding_db.t_order" db-types="MySQL" />
+    <sql-case id="show_index_with_table_back_quotes" value="SHOW INDEXES FROM `t_order` FROM sharding_db" db-types="MySQL" />
+    <sql-case id="show_index_with_database_back_quotes" value="SHOW INDEXES FROM t_order FROM `sharding_db`" db-types="MySQL" />
+    <sql-case id="show_index_with_back_quotes" value="SHOW KEYS FROM `sharding_db`.`t_order`" db-types="MySQL" />
+    <sql-case id="show_all" value="SHOW ALL" db-types="PostgreSQL" />
+    <sql-case id="show_server_version" value="SHOW SERVER_VERSION" db-types="PostgreSQL" />
```

Comment:
sql-case of db-types="PostgreSQL" should be removed, test for visitor is based on MySQL.